cmd/root.go

```diff
@@ -243,6 +243,11 @@ func init() {
 	flags.String("workdir", "", "path to a Supabase project directory")
 	flags.Bool("experimental", false, "enable experimental features")
 	flags.String("network-id", "", "use the specified docker network instead of a generated one")
+	var outputFlag = utils.EnumFlag{
+		Allowed: append([]string{utils.OutputEnv}, utils.OutputDefaultAllowed...),
+		Value:   utils.OutputPretty,
+	}
+	flags.Var(&outputFlag, "output", "output format of status variables")
 	flags.Var(&utils.DNSResolver, "dns-resolver", "lookup domain names using the specified resolver")
```

Comment:
Should I introduce another util like you did with `DNSResolver`?

Reply:
I added `utils.DNSResolver` to primarily avoid cyclic imports from utils => cmd package. And yes, I think it makes changes simpler if we do the same for `utils.OutputFormat`.

For eg. you won't need to change every function signature to accept a format arg, because you can compare `utils.OutputFormat == utils.OutputPretty` directly in internal packages.

Reply:
Done